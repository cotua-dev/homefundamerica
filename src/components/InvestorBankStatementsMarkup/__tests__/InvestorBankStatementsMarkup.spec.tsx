import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as InvestorBankStatementsMarkup } from '..';

describe('InvestorBankStatementsMarkup component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<InvestorBankStatementsMarkup/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
