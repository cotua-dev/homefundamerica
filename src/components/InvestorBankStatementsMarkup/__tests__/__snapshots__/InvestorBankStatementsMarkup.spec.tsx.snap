// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvestorBankStatementsMarkup component renders 1`] = `
<section
  class="contentWrapper"
>
  <article>
    <h2>
      What is an Investor Bank Statement Loan?
    </h2>
    <strong>
      In short, we use 2 years of personal or business bank statements and a P & L (profit and loss) statement to verify proof of income, instead of tax returns.
    </strong>
    <p>
      MBANC understands what it means to own a business.
                We know that owning a small business, start-up, or
                having another form of untraditional income makes
                homeownership a more difficult feat. That's why we
                offer this flexible mortgage program to sidestep some
                of those challenges and create a clear path to ownership.
    </p>
    <h2>
      Qualification Criteria
    </h2>
    <p>
      <strong>
        85% LTV
      </strong>
       up to a loan amount of $1,500,000.00 with a 680 FICO score (SFR Only)
    </p>
    <p>
      <strong>
        70% LTV
      </strong>
       Rate and Term and Unlimited Cash Out refinances available
    </p>
    <p>
      <strong>
        Reserve requirements
      </strong>
       are 6 months PITI up to a $1,500,000.00 Loan amount, anything above requires 12 months PITI
    </p>
    <p>
      <strong>
        Cash Out
      </strong>
       can be used as reserves
    </p>
    <p>
      <strong>
        Cash Vest
      </strong>
       in LLC or Corporation
    </p>
    <p>
      Apply Now For True Stated Income
    </p>
  </article>
</section>
`;
